extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/ControllerImpl.java

```diff
@@ -1187,7 +1187,7 @@ private Yielder<Object[]> getFinalResultsYielder(
 
       final InputChannelFactory inputChannelFactory;
 
-      if (MultiStageQueryContext.isDurableStorageEnabled(task.getQuerySpec().getQuery().context())) {
+      if (MultiStageQueryContext.isDurableShuffleStorageEnabled(task.getQuerySpec().getQuery().context())) {
```

Comment:
nit: While the changes are not made in this PR, I think we can reuse `isDurableStorageEnabled` instead of this method call again.

Reply:
done, thanks for the suggestion